### ISSUE TITLE
Exception in thread "main" com.intel.analytics.bigdl.nn.abstractnn.InvalidLayer

### ISSUE THREAD

**Issue by land1725** [intel-analytics/BigDL#2698]
Hello,when I use Dense Layer,some errors occur ,could you help me?

Engine.init

val hidden_1 = Dense(128, activation = "relu", inputShape = Shape(inputNumber))
val hidden_2 = Dense(128, activation = "relu")
val hidden_3 = Dense(1, activation = "sigmoid")

val model = Sequential()
  .add(hidden_1)
  .add(hidden_2)
  .add(hidden_3)

val criterion = BCECriterion()
val estimator = new DLClassifier(model, criterion, Array(inputNumber))
  .setFeaturesCol("features")
  .setLabelCol("label")
  .setBatchSize(1280)
  .setLearningRate(0.01)
  .setOptimMethod(new Adam[Float]())
  .setValidation(Trigger.everyEpoch, testInput, Array(new AUC()), 3096)
val dlModel = estimator.fit(trainInput.coalesce(4))
val testRank = dlModel.transform(testInput.coalesce(4))
testRank.show(100, false)
18/12/03 20:52:44 INFO utils.Engine$: Find existing spark context. Checking the spark conf...
Exception in thread "main" com.intel.analytics.bigdl.nn.abstractnn.InvalidLayer: Do not mix Sequential[be57a7d7]{
[input -> -> output]

}(isKerasStyle=false) with Layer
(isKerasStyle=true):
Dense[e5d3c02e]
at com.intel.analytics.bigdl.nn.abstractnn.InferShape$class.excludeInvalidLayers(InferShape.scala:98)
at com.intel.analytics.bigdl.nn.abstractnn.AbstractModule.excludeInvalidLayers(AbstractModule.scala:58)
at com.intel.analytics.bigdl.nn.abstractnn.InferShape$class.validateInput(InferShape.scala:108)
at com.intel.analytics.bigdl.nn.abstractnn.AbstractModule.validateInput(AbstractModule.scala:58)
at com.intel.analytics.bigdl.nn.DynamicContainer.add(DynamicContainer.scala:44)
at HuoyingBigDL$.main(HuoyingBigDL.scala:686)
at HuoyingBigDL.main(HuoyingBigDL.scala)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
at org.apache.spark.deploy.SparkSubmit$.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:894)
at org.apache.spark.deploy.SparkSubmit$.doRunMain$1(SparkSubmit.scala:198)
at org.apache.spark.deploy.SparkSubmit$.submit(SparkSubmit.scala:228)
at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:137)
at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
18/12/03 20:52:44 INFO spark.SparkContext: Invoking stop() from shutdown hook

**Comment by hkvision** [comment]
Seems you have imported the wrong Sequential, you are suggested to import the following:
```
import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
import com.intel.analytics.zoo.pipeline.api.keras.layers._
```

**Comment by land1725** [comment]
import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
import com.intel.analytics.zoo.pipeline.api.keras.layers._
the problem has been solved

**Comment by land1725** [comment]
@hkvision hello,I am a little confused about Sequential imported.
I did not see some documents about com.intel.analytics.zoo.pipeline.api.keras.models.Sequential and com.intel.analytics.zoo.pipeline.api.keras.layers._ .But as you said ,it solved my porblem.

Could you help me to check the other import whether correct?

import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric.NumericFloat
import com.intel.analytics.bigdl.utils.Engine
import com.intel.analytics.bigdl.nn._
import com.intel.analytics.bigdl.optim.Adam
import com.intel.analytics.bigdl.dlframes.{DLClassifier, DLModel}
import com.intel.analytics.bigdl.optim.Trigger
import com.intel.analytics.bigdl.utils.Shape

import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
import com.intel.analytics.zoo.pipeline.api.keras.layers._
import com.intel.analytics.zoo.pipeline.api.keras.metrics.AUC

Engine.init

    val hidden_1 = Dense(128, activation = "relu", inputShape = Shape(inputNumber))
    val hidden_2 = Dense(128, activation = "relu")
    val hidden_3 = Dense(1, activation = "sigmoid")

    val model = Sequential()
      .add(hidden_1)
      .add(hidden_2)
      .add(hidden_3)

    val criterion = BCECriterion()
    val estimator = new DLClassifier(model, criterion, Array(inputNumber))
      .setFeaturesCol("scaledFeatures")
      .setLabelCol("label")
      .setBatchSize(1280)
      .setLearningRate(1e-3)
      .setMaxEpoch(10)
      .setOptimMethod(new Adam[Float]())
      .setValidation(Trigger.everyEpoch, testInput, Array(new AUC()), 1280)
    val dlModel = estimator.fit(trainInput.coalesce(4))
    val testRank = dlModel.transform(testInput.coalesce(4))
    testRank.show(100, false)

**Comment by hkvision** [comment]
@land1725 I checked that your other imports should be correct.
For the documentation of Analytics Zoo Keras-style layers, we are still working in progress :)
